index.html

```diff
@@ -234,52 +287,50 @@ <h1 class="about_myself_heading">About me</h1>
       </ul>
     </section>
 
-    <section class="contact_section">
-      <h3>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h3>
-
-      <form
-        action="https://formspree.io/f/xgedowvw"
-        method="POST"
-        class="contact_form"
-      >
-        <input
-          class="name"
-          type="text"
-          placeholder="Full name"
-          required
-          maxlength="30"
-        />
-        <hr class="horizontal_line" />
-        <input
-          class="email"
-          type="email"
-          placeholder="Email address"
-          required
-        />
-        <hr class="horizontal_line" />
-        <textarea
-          class="message"
-          name="message"
-          id="message"
-          cols="30"
-          rows="10"
-          maxlength="500"
-          placeholder="Enter text here"
-          required
-        ></textarea>
-
-        <button class="submit_button" type="submit">Get in touch</button>
-      </form>
-    </section>
-
-    <hr class="horizontal_line horizontal_line_3" />
-
     <footer class="footer">
-      <div class="social-media social-media-footer">
-        <ul class="social-media-links">
+      <section class="contact_section" id="contact">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form
+          action="https://formspree.io/f/xgedowvw"
+          method="POST"
+          class="contact_form"
+        >
+          <input
+            class="full_name"
+            type="text"
+            placeholder="Full Name"
+            name="full_name"
+            required
+            maxlength="30"
+          />
+
+          <input
+            class="email"
+            type="email"
+            placeholder="Email address"
+            required
+          />
+
+          <textarea
+            class="message"
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            placeholder="Enter text here"
+            required
```

Comment:
- [ ] OPTIONAL: I can see you did a good job there... But you went too far and this even introduced a horizontal scroll overflow! 

![image](https://user-images.githubusercontent.com/57421823/149513953-43cec9c1-e90c-443b-93f4-3f89ffdb52e7.png)


Just please kindly I suggest you correct it.  (I can see you implemented a `width: 150%` on each element.  I tested reducing it to 130% and it worked wonderfully!!!). 😄

Reply:
Thank you very much for helping me to make my project looks similar to the template. 👍